README.md

```diff
@@ -17,18 +17,18 @@ All applications should be running on their default ports.
 
 | App name          | Address (HTTP)                                       | Address (HTTPS)                                           |
 |-------------------|------------------------------------------------------|-----------------------------------------------------------|
-| node_exporter     | [demo.do.prometheus.io:9100][node_exporter_http]     | [node.demo.do.prometheus.io][node_exporter_https]         |
-| prometheus        | [demo.do.prometheus.io:9090][prometheus_http]        | [prometheus.demo.do.prometheus.io][prometheus_https]      |
-| alertmanager      | [demo.do.prometheus.io:9093][alertmanager_http]      | [alertmanager.demo.do.prometheus.io][alertmanager_https]  |
-| grafana           | [demo.do.prometheus.io:3000][grafana_http]           | [grafana.demo.do.prometheus.io][grafana_https]            |
+| node_exporter     | [demo.prometheus.io:9100][node_exporter_http]     | [node.demo.prometheus.io][node_exporter_https]         |
+| prometheus        | [demo.prometheus.io:9090][prometheus_http]        | [prometheus.demo.prometheus.io][prometheus_https]      |
+| alertmanager      | [demo.prometheus.io:9093][alertmanager_http]      | [alertmanager.demo.prometheus.io][alertmanager_https]  |
+| grafana           | [demo.prometheus.io:3000][grafana_http]           | [grafana.demo.prometheus.io][grafana_https]            |
 
 ## Important notice
 
 Before running, golang is required to be installed on deployer machine (necessary to install random_exporter).
 
 Most services can be accessed in two ways (links in [Applications](#Applications) section. As an example, prometheus can be accessed via:
-  - **http**://demo.do.prometheus.io:9090 - default way
-  - **https**://prometheus.do.prometheus.io - workaround which in background communicates with prometheus via insecure, "default" channel mentioned above
+  - **http**://demo.prometheus.io:9090 - default way
+  - **https**://prometheus.prometheus.io - workaround which in background communicates with prometheus via insecure, "default" channel mentioned above
```

Comment:
Shouldn't this be prometheus.demo.prometheus.io instead of prometheus.prometheus.io ?

Reply:
Yes, fixed.